Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && \
+    apt-get install -y wget \
+                       curl \
```

Comment:
If you already have `wget`, why do you need `curl`?

Reply:
:(